### PR TITLE
refactor: refactor config to handle diff inbound/outbound certs

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,22 +52,85 @@ This package exemplifies the use of the Mojaloop SDK Standard Components for TLS
...
+   and they handle the rest.
 
-### Outbound API
-  `Outbound API` service is used by `Thirdparty backend` to make a call to `Mojaloop Switch`  
-  Its responsibility is to transform asynchronous Mojaloop API native interface's set of calls to a synchronous call.
+   config.INBOUND.TLS represents the certs needed by the `thirdparty-sdk` Hapi Inbound server.
 
+
+### When used by a DFSP
+   DFSP's needs to have Inbound service running, DFSP's will need to implement
```
Comment (elnyry-sam-k) [inline README.md:99]:
```suggestion
   DFSPs need to have Inbound service running, DFSPs will need to implement
```

File: README.md [hunk 2]
```diff
@@ -52,22 +52,85 @@ This package exemplifies the use of the Mojaloop SDK Standard Components for TLS
...
+#### Inbound API
+   `Inbound API` service is called by `Mojaloop Switch`.
+   Its responsibility is to forward calls to `Thirdparty Backend` or help to deliver synchronous response for calls initiated by `Thirdparty backend` on `Outbound API`
+
+##### Deep Dive
+   Inbound handlers handle receiving of requests coming from the Switch.
+   These typically publish responses into redis for state machines to continue a flow.
+
+   Inbound models handle the processing of Switch requests but they themselves
+   can make requests to the accomplish a certain flow the initial inbound request initiates.
```
Comment (mdebarros) [inline README.md:73]: Is there any reason to manually word wrap this? <-- comment applies through-out the readme changes.

File: src/shared/config.ts [hunk 3]
```diff
@@ -167,6 +180,30 @@ export const ConvictConfig = Convict<ServiceConfig>({
...
+      },
+      creds: {
+        ca: '',
+        cert: '',
+        key: ''
+      }
+    }
+  },
+  CONTROL: {
+    MGMT_API_WS_URL: {
```
Comment (mdebarros) [inline src/shared/config.ts:196]: Do we need any TLS config for MGM API?

**Review by mdebarros — APPROVED** [review]
Looks ok to me.

Just some minor comments for consideration.

File: src/shared/config.ts (hunk 3 above)
Comment (kleyow) [reply src/shared/config.ts:196]: Not that I know of based off of looking at `mojaloop-connector`

**Pull request merged**